### ISSUE TITLE
Update `email_account` resource to `connected_account`

### ISSUE THREAD

**Issue by AlecRosenbaum** [taylorbrooks/closeio#63]
Hi!

I noticed that this gem has a resource for [accessing `email_account`](https://github.com/taylorbrooks/closeio/blob/c53b8a717a0fd55e579672753ebe59a6fb7a3840/lib/closeio/resources/email_account.rb#L5) on the api, and I can also see some traffic on that endpoint from this gem. Would it be possible to update or replace the resource to hit [`connected_account`](https://developer.close.com/#connected-accounts) instead?

I'm not sure if `email_account` was documented at one point, but internally we've been moving away from it in favor of `connected_account` for some time now. The connected account endpoint should be better documented, and will be better supported going forward!

I'm not very experienced with ruby, but if it's as simple as `s/email/connected/g` within the ruby code I can submit a patch.

**Comment by taylorbrooks** [comment]
@AlecRosenbaum Is the response structure the same?

**Comment by squeaky-pl** [comment]
@taylorbrooks Yes, you get exactly the same fields you would get from `email_account`. We are planning on deprecating and removing `email_account` soon as most of our customers migrated away from it.

**Comment by neob91-close** [comment]
@taylorbrooks We'd really appreciate any help you can give us to push this forward.

**Comment by taylorbrooks** [comment]
@neob91-close I'm happy to accept PRs!

**Comment by neob91-close** [comment]
Opened a PR: https://github.com/taylorbrooks/closeio/pull/83.

I have 0 experience with Ruby though, so I could really use your help in making sure that the change works okay.
